lnwire/features.go

```diff
@@ -155,6 +155,16 @@ const (
 	// TODO: Decide on actual feature bit value.
 	ExplicitChannelTypeOptional = 45
 
+	// PaymentMetadataRequired is a required bit that denotes that an
+	// invoice contains metadata that must be passed along with the payment
```

Comment:
nit: phrasing of comment isn't super clear to me here 

"denotes that if an invoice contains metadata, it must be passed along ..."?

Reply:
change applied